Rakefile

```diff
@@ -780,4 +780,24 @@ task update_all: [:run_all, :post_process_all, :visualize_all] do
   # run and post_process all the scenarios
 end
 
+desc 'Run residential tasks'
+namespace :residential do
+  desc 'Update residential resources'
```

Comment:
Thank you for making this rake task! This will reduce my anxiety on this step a lot. 🙌